test/ui-testing/loan_renewal.js

```diff
@@ -527,8 +525,8 @@ module.exports.test = (uiTestCtx, nightmareX) => {
               .click('#clickable-edit-item')
               .wait('#clickable-delete-entry')
               .click('#clickable-delete-entry')
-              .wait('#clickable-delete-item-confirmation-confirm')
-              .click('#clickable-delete-item-confirmation-confirm')
+              .wait('[data-test-confirmation-modal-confirm-button]')
```

Comment:
The `[data-test...]` elements will be stripped from the production build and therefor unavailable when the integration tests run against folio-testing or folio-snapshot.